test/models/orchestration/compute_test.rb

```diff
@@ -23,6 +23,15 @@ class ComputeOrchestrationTest < ActiveSupport::TestCase
       assert(@host.send :setComputeDetails)
     end
 
+    test "are set for CR providing IPv6" do
+      an_ip6 = '2001:db8::1'
+      @cr.stubs(:provided_attributes).returns({:ip6 => :ip6})
+      @host.vm.expects(:ip6).returns(an_ip6)
```

Comment:
A test that returns a blank value for :ip6 while claiming to provide ip4 + ip6 would be a good idea.